LICENSE

```diff
@@ -0,0 +1,9 @@
+Copyright (c) 2024 Tweede Golf, and Contributors
+
+Except as otherwise noted (below and/or in individual files), drive-rs is
+licensed under the MIT license <LICENSE-MIT> or <http://opensource.org/licenses/MIT>, at your option.
```

Comment:
```suggestion
Except as noted below, drive-rs is licensed under the MIT license <LICENSE_MIT>.
```

This old phrasing is confusing, since it is often used in Rust project to introduce the Apache+MIT dual licensing; it suggests that there are two licenses the user can pick from: MIT or MIT.